planner/core/logical_plan_builder.go

```diff
@@ -3173,15 +3176,38 @@ func unfoldWildStar(field *ast.SelectField, outputName types.NameSlice, column [
 	return resultList
 }
 
-func (b *PlanBuilder) addAliasName(selectFields []*ast.SelectField, p LogicalPlan) (resultList []*ast.SelectField, err error) {
-	if len(selectFields) != len(p.OutputNames()) {
```

Comment:
This error should be left instead of being removed directly?

Reply:
Before this PR, `addAliasName` assumes one to one mapping between sel.Fields.Fields and p.OutputNames().
So we need this check.

After this PR, we only iterate over selectFields. So no need to check this.